src/Illuminate/Support/helpers.php

```diff
@@ -314,12 +314,14 @@ function tap($value, $callback = null)
     /**
      * Throw the given exception if the given condition is true.
      *
+     * @template TException of \Throwable
+     *
      * @param  mixed  $condition
-     * @param  \Throwable|string  $exception
+     * @param  TException|class-string<TException>  $exception
```

Comment:
Type hint does not fully define the parameter options; reading the code below; the type hit for this param should be: `@param  TException|class-string<TException>|string`
If the provided string is not a class name the string is passed in the `new RuntimeException($exception)` message param

Reply:
Already fixed by another PR.